subsys/debug/cpu_load/Kconfig

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menuconfig CPU_LOAD
+	bool "Enable CPU load measurement"
+	select NRFX_PPI if HAS_HW_NRF_PPI
+	select NRFX_DPPI if HAS_HW_NRF_DPPIC
+	depends on !SOC_SERIES_NRF51X #Lack of required HW events
+	help
+	  Enable CPU load measurement instrumentation. Tool is using one TIMER
+	  peripheral and PPI to perfrom accurate CPU load measurement.
+
+if CPU_LOAD
+
+module = CPU_LOAD
+module-str = CPU load measurement
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+config CPU_LOAD_CMDS
+	bool "Enable shell commands"
+	depends on SHELL
+	default y
+
+config CPU_LOAD_LOG_PERIODIC
+	bool "Periodically log current CPU load"
+	help
+	  INFO level must be enabled to get the log.
+
+config CPU_LOAD_LOG_INTERVAL
+	int "Logging interval for CPU load [ms]"
+	depends on CPU_LOAD_LOG_PERIODIC
+	default 2000
+
+config CPU_LOAD_ALIGNED_CLOCKS
+	bool "Enable aligned clock sources"
+	help
+	  If enabled then sleep period measurement is done using the same clock
```

Comment:
```
After enabling this option, the sleep period measurement
is done using the same clock source as the active period,
and one additional PPI channel is used.
If you use the internal RC oscillator for at least one
of the clock sources (high or low frequency), this option
will ensure more accurate results.
Enabling this option allows going to low power idle mode
because the high frequency clock is not used by this module.
```